### DISCUSSION TITLE
Is there a way to hide multiple elements at once?

### DISCUSSION THREAD

**Discussion by NatSavage** [MyreMylar/pygame_gui#297] [Q&A] [answered]
I was wondering if there is a way to easily show and hide UI elements as a group. I tried setting visible on a container to 0, hoping that it would hide the elements within but it only hid the container border. Is there a way to mass show/hide elements in the current version?

**Comment by MyreMylar** [comment, marked as answer]
`container.hide()`  (and `container.show()`) should do it since version 0.5.7

**Comment by NatSavage** [comment]
tysm! I hadnt seen it. :)
